docs/en/sql-reference/statements/create/user.md

```diff
@@ -52,33 +53,50 @@ Another way of specifying host is to use `@` syntax following the username. Exam
 
 !!! info "Warning"
     ClickHouse treats `user_name@'address'` as a username as a whole. Thus, technically you can create multiple users with the same `user_name` and different constructions after `@`. However, we don’t recommend to do so.
+	
+## GRANTEES Clause {#grantees}
+
+Specifies users or roles which are allowed to receive [privileges](../../../sql-reference/statements/grant.md#grant-privileges) from this user on the condition this user has also all required access granted with [GRANT](../../../sql-reference/statements/grant.md) option. Options of the `GRANTEES` clause:
+
+-   `user` — Specifies a user this user can grant privileges to.
+-   `role` — Specifies a role this user can grant privileges to.
+-   `ANY` — This user can grant privileges to anyone. It's the default setting.
+-   `NONE` — This user can grant privileges to none.
+
+You can exclude any user or role by using the `EXCEPT` expression. For example, `CREATE USER user1 GRANTEES ANY EXCEPT user2`. It means if `user1` has some privileges granted with the `GRANT` option it will be able to grant those privileges to anyone except `user2`.
```

Comment:
Please replace it with

... granted with `GRANT OPTION` it will ...

`GRANT OPTION` is a keyword, see https://clickhouse.tech/docs/en/sql-reference/statements/grant/#grant-privigele-syntax

That link would be also useful in the documentation here.

Reply:
> Please replace it with
> 
> ... granted with `GRANT OPTION` it will ...
> 
> `GRANT OPTION` is a keyword, see https://clickhouse.tech/docs/en/sql-reference/statements/grant/#grant-privigele-syntax
> 
> That link would be also useful in the documentation here.

Ok, only WITH GRANT OPTION.